lib/Models/UserDrawing.js

```diff
@@ -157,7 +157,9 @@ var UserDrawing = function(options) {
     // helper for dragging points around
     var that = this;
     this.dragHelper = new DragPoints(options.terria, function(customDataSource) {
-        that.onPointMoved(customDataSource);
+        if (typeof that.onPointMoved === "function") {
```

Comment:
Sure, will change. From what I understand, javascript allows both interchangeably though... is there a style guide I should be following?